x-pack/plugins/canvas/public/components/es_data_view_select/es_data_view_select.component.tsx

```diff
@@ -8,27 +8,27 @@
 import React, { FocusEventHandler } from 'react';
 import { EuiComboBox } from '@elastic/eui';
 
-export interface ESIndexSelectProps {
+export interface ESDataViewSelectProps {
   loading: boolean;
   value: string;
-  indices: string[];
-  onChange: (index: string) => void;
+  dataViews: string[];
```

Comment:
How are dataViews represented by strings? Just making sure will work in all circumstances.